### ISSUE TITLE
function `std.manifestYamlDoc` has no parameter `quote_keys`

### ISSUE THREAD

**Issue by WoozyMasta** [jsonnet-libs/docsonnet#29]
Hello!

I ran into a problem the `std.manifestYamlDoc` function does not have a `quote_keys` parameter

Apparently because the project uses an older version of jsonnet 0.15
https://github.com/jsonnet-libs/docsonnet/blob/be066653f33fcaa8467231f6334f6878b92fd8f9/go.mod#L8

Are there any restrictions for upgrading to jsonnet 0.18?

---

Below is an example of how I discovered the issue:

```jsonnet
  '#new'+: d.fn(
    help=
    |||
      Some function

      Optional arguments can be passed as an array of `args`

      ```yaml
      %(defaults)s
      ```
    ||| % { defaults: std.manifestYamlDoc({args: defaults}, quote_keys=false) },
    args=[
      d.arg('name', d.T.string),
      d.arg('args', d.T.array, {}),
    ]
  ),
  new(name=error 'Application name is unset', args={})::
```

```bash
docsonnet -o docs/ main.libsonnet
```

```
Extracting: RUNTIME ERROR: function has no parameter quote_keys
```

Also I tried using docsonnet versions:

* `v0.0.4-0.20220713111504-be066653f33f`
*  `v0.0.3`

**Comment by WoozyMasta** [comment]
Perhaps the question arises why I do not use `jsonnet -S -c -m docs/ docs.jsonnet`, but do documentation through `docsonnet`

Because, for example, the [jsonnet-libs/k8s](https://github.com/jsonnet-libs/k8s) project still [uses](https://github.com/jsonnet-libs/k8s/blob/9a9ca58385dc0c5ed9b33f72ddecdfbf8a6f10df/scripts/docs.sh#L19) `docsonnet` and I couldn't fix documentation generation with `doc-util/render.libsonnet`

Here I am getting an error:
```
RUNTIME ERROR: Unexpected type string, expected object
```
https://github.com/jsonnet-libs/docsonnet/blob/be066653f33fcaa8467231f6334f6878b92fd8f9/doc-util/render.libsonnet#L251


Hanging a few `std.trace`, I realized that the problem is in the `ignore` argument
Specifically here:
https://github.com/jsonnet-libs/k8s-libsonnet/blob/e22027656445547ff5e757c10a3cb7d8c933a441/1.24/_gen/admissionregistration/v1/mutatingWebhook.libsonnet#L64

I would suggest a fix for `doc-util/render.libsonnet` but I have just started learning jsonnet and so far the solution to the problem is not obvious to me.

**Comment by Duologic** [comment]
The updated golang has been merged and lint issues fixed, I've pushed a new tag v0.0.4 and this made a new docker image. I'd consider this issue done.

Additionally the 'Unexpected type' error in the jsonnet renderer is fixed too.
